react-native-plaid-link-sdk.d.ts

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import { TouchableOpacity, TouchableOpacityProps } from 'react-native';
+
+export interface Account {
+  id: string;
+  mask: string;
+  name: string;
+  subtype: string;
+  type: string;
+}
+
+export interface Institution {
+  institution_id: string;
+  name: string;
+}
+
+export interface OnSuccessMetadata {
+  status: string;
+  link_session_id: string;
+  request_id: string;
+  public_token: string;
+  account_id: string;
+  institution: Institution;
+  accounts: Array<Account & {
+    verification_status?: {
+      pending_automatic_verification: boolean;
+      pending_manual_verification: boolean;
+      manually_verified: boolean;
+    };
+  }>;
+  account: Account
+}
+
+export type ErrorStatus =
+  | 'requires_questions'
+  | 'requires_selections'
+  | 'requires_code'
+  | 'choose_device'
+  | 'requires_credentials'
+  | 'institution_not_found';
+
+export interface OnExitMetadata {
+  link_session_id: string;
+  request_id: string;
+  status: ErrorStatus;
+  institution: Institution;
+}
+
+export type EventName =
+  | 'ERROR'
+  | 'EXIT'
+  | 'HANDOFF'
+  | 'OPEN'
+  | 'OPEN_MY_PLAID'
+  | 'SEARCH_INSTITUTION'
+  | 'SELECT_INSTITUTION'
+  | 'SUBMIT_CREDENTIALS'
+  | 'SUBMIT_MFA'
+  | 'TRANSITION_VIEW';
+
+export type ViewName =
+  | 'CONNECTED'
+  | 'CONSENT'
+  | 'CREDENTIAL'
+  | 'ERROR'
+  | 'EXIT'
+  | 'LOADING'
+  | 'MFA'
+  | 'NUMBERS'
+  | 'RECAPTCHA'
+  | 'SELECT_ACCOUNT'
+  | 'SELECT_INSTITUTION';
+
+export interface EventMetadata {
+  error_code: string;
+  error_message: string;
+  error_type: string;
+  exit_status: string;
+  institution_id: string;
+  institution_name: string;
+  institution_search_query: string;
+  request_id: string;
+  link_session_id: string;
+  mfa_type: string;
+  view_name: ViewName;
+  timestamp: string;
+}
+
+export interface OnEventArgs {
+  eventName: EventName;
+  metadata: EventMetadata;
+}
```

Comment:
are you getting these? I just get an object with `event` and `metadata` as properties.